src/Cid/Cid.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+   <metadata>
+       <id>Cid</id>
+       <version>$version$</version>
+       <title>Xamarin Android USB peripherals</title>
+       <authors>Syncromatics Development</authors>
```

Comment:
Let's use Syncromatics Engineering to stay in line with the "This software is made available by Syncromatics Engineering under the MIT license." line from README.md